src/common/variables.h

```diff
@@ -61,6 +61,8 @@ char *dt_variables_expand(dt_variables_params_t *params, gchar *source, gboolean
 /** reset sequence number */
 void dt_variables_reset_sequence(dt_variables_params_t *params);
 
+char *dt_format_exposure(float exposuretime);
```

Comment:
As you see the convention is to use dt_variables_... in variables.h.

I also feel that such common routine should not be declared here. Maybe in utility.h ?

`dt_util_format_exposure(const float exposuretime);`

Or maybe another proposal ?